### ISSUE TITLE
NVDA not reading un-labelled graphics on the Bloomberg Messaging Console

### ISSUE THREAD

**Issue by EneyeAttah** [nvaccess/nvda#17312]
### Steps to reproduce:
NVDA not reading un-labelled graphics on the Bloomberg Messaging Console

### Actual behavior:
Nvda does not read and identify un-labelled graphics when navigating Bloomberg Messaging console.
It does not read the unlabelled graphics

### Expected behavior:
Sound should be made when navigating the Bloomberg Messaging console via NVDA
Jaws makes sound while navigating the Messaging console and identifies un-labelled graphics

### NVDA logs, crash dumps and other attachments:
None

### System configuration
NVDA Installed

#### NVDA version:
2024.3.1 (2024.3.1.33894)

#### Windows version:
Windows 11 Pro Version 22H2 dev ( build 22621.4317)

#### Name and version of other software in use when reproducing the issue:
Bloomberg Messaging console.

#### Other information about your system:
Bloomberg Messaging console is a collaboration Tool that is used to send instant messages across chat rooms

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Same behavior with earlier version.

#### If NVDA add-ons are disabled, is your problem still occurring?
NA

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
NA

**Comment by Adriani90** [comment]
Can you please upload a log file with logging set to debug in NVDA general settings?
After setting log level to debug, restart NVDA and go to Bloomberg, and move with the keyboard to an unlabelled grafic.
Then, right after that, press NVDA key +f1 and save that file.
Please upload the log file to this issue.

**Comment by seanbudd** [comment]
This is currently intended behaviour in NVDA. We do not have inherent support for unlabelled graphics. Please report this issue to the Bloomberg developers.

**Comment by Adriani90** [comment]
Closing as duplicate of #7412.
